test/test_jit_fuser_te.py

```diff
@@ -1236,6 +1236,31 @@ def fn(x):
             self.assertEqual(ref, t(x))
             self.assertEqual(len(self.findFusionGroups(t.graph_for(x))), 0)
 
+    @unittest.skipIf(not RUN_CUDA, "fuser requires CUDA")
+    def test_superslomo(self):
+        # Test extracted from Super-SloMo: https://github.com/avinashpaliwal/Super-SloMo
```

Comment:
Would you mind adding a brief comment explaining what the problem was? In a month noone would remember what this was about and the test would (hopefully) be always passing - it would be easy to think that it doesn't provide any useful coverage and it would be tempting to remove it.